package.json

```diff
@@ -20,13 +20,12 @@
     "test": "test"
   },
   "scripts": {
-    "prebuild": "mkdir -p build",
-    "build": "browserify src/vl.ts -p tsify -g [ babelify ] -d -s vl | exorcist build/vega-lite.js.map > build/vega-lite.js && cp package.json build",
-    "postbuild": "uglifyjs build/vega-lite.js -cm --in-source-map build/vega-lite.js.map --source-map build/vega-lite.min.js.map > build/vega-lite.min.js && tsc && npm run schema",
-    "build:all": "npm run clean && npm run build && npm run test:noschema && npm run build:images",
+    "prebuild": "mkdir -p build/site build/examples/images",
+    "build": "tsc && cp package.json build && browserify build/src/vl.js -g [ babelify ] -d -s vl | exorcist build/vega-lite.js.map > build/vega-lite.js",
```

Comment:
Removing `tsify` make sourcemap no longer works for browserify bundle (e.g., in editor)